WeakAuras/SubRegionTypes/Glow.lua

```diff
@@ -0,0 +1,319 @@
+    elseif newType == "Pixel" then
+      self.glowStart = LCG.PixelGlow_Start
+      self.glowStop = LCG.PixelGlow_Stop
+    end
+    self.glowType = newType
+    if isGlowing then
+      self:SetVisible(true)
+    end
+  end,
+  SetUseGlowColor = function(self, useGlowColor)
+    self.useGlowColor = useGlowColor
+    if self.glow then
+      self:SetVisible(true)
+    end
+  end,
+  SetGlowColor = function(self, r, g, b, a)
+    self.glowColor = {r, g, b, a}
+    if self.glow then
+      self:SetVisible(true)
+    end
+  end,
+  SetGlowLines = function(self, lines)
+    self.glowLines = lines
+    if self.glow then
+      if self.glowType == "ACShine" then -- workaround ACShine not updating numbers of dots
+        self:SetVisible(false)
+      end
+      self:SetVisible(true)
+    end
+  end,
+  SetGlowFrequency = function(self, frequency)
+    self.glowFrequency = frequency
+    if self.glow then
+      self:SetVisible(true)
+    end
+  end,
+  SetGlowLength = function(self, length)
+    self.glowLength = length
+    if self.glow then
+      self:SetVisible(true)
+    end
+  end,
+  SetGlowThickness = function(self, thickness)
+    self.glowThickness = thickness
+    if self.glow then
+      self:SetVisible(true)
+    end
+  end,
+  SetGlowScale = function(self, scale)
+    self.glowScale = scale
+    if self.glow then
+      self:SetVisible(true)
+    end
+  end,
+  SetGlowBorder = function(self, border)
+    self.glowBorder = border
+    if self.glow then
+      self:SetVisible(true)
+    end
+  end,
+  SetGlowXOffset = function(self, xoffset)
+    self.glowXOffset = xoffset
+    if self.glow then
+      self:SetVisible(true)
+    end
+  end,
+  SetGlowYOffset = function(self, yoffset)
+    self.glowYOffset = yoffset
+    if self.glow then
+      self:SetVisible(true)
+    end
+  end
+}
+
+local function create()
+  local region = CreateFrame("FRAME", nil, UIParent)
+
+  for name, func  in pairs(funcs) do
+    region[name] = func
+  end
+
+  return region
+end
+
+local function onAcquire(subRegion)
+  subRegion:Show()
+end
+
+local function onRelease(subRegion)
+  subRegion:Hide()
+end
+
+
+
+local function modify(parent, region, parentData, data, first)
+  region:SetParent(parent)
+  parent:AnchorSubRegion(region, "area", parentData.regionType == "aurabar" and data.glow_anchor)
+
+  region.parentType = parentData.regionType
+  region.useGlowColor = data.useGlowColor
+  region.glowColor = data.glowColor
+  region.glowLines = data.glowLines
+  region.glowFrequency = data.glowFrequency
+  region.glowLength = data.glowLength
+  region.glowThickness = data.glowThickness
+  region.glowScale = data.glowScale
+  region.glowBorder = data.glowBorder
+  region.glowXOffset = data.glowXOffset
+  region.glowYOffset = data.glowYOffset
+
+  region:SetGlowType(data.glowType)
+  region:SetVisible(data.glow)
+
+  region.Update = function() end
+end
+
+local function supports(regionType)
+  return regionType == "icon"
+         or regionType == "aurabar"
```

Comment:
is there an interest to add support for "text" too?

Reply:
Well, it's pretty easy to add that, but I think it would look very ugly. We cab